libsolidity/formal/SolverInterface.h

```diff
@@ -69,6 +69,21 @@ class Expression
 		});
 	}
 
+	static Expression implies(Expression _a, Expression _b)
+	{
+		return !_a || _b;
+	}
+
+	static Expression lTrue()
```

Comment:
You should be able to use `Expression(true)` instead of `lTrue()`.

Reply:
In `smt::Expression` yes, but won't that become an `Integer` variable inside `z3`?

Reply:
I think this is something we have to fix anyway.

Reply:
If it only works like this, please leave it as it is but add a comment that this should work the same way as `Expression(true)`.

Reply:
My assumption was wrong, it does work as Boolean. Removing lTrue and lFalse.